.swiftlint.yml

```diff
@@ -5,6 +5,12 @@ disabled_rules:
   - trailing_whitespace # Disables SwiftLint complaining about whitespace characters on empty lines
   - todo # Disables auto-warning of TODO statements
 
+nesting:
```

Comment:
Now that we have this `nesting` exception in `.swiftlint.yml`, we can remove the `swiftlint:disable` and `swiftlint:enable` comments out of `HTTP.swift`.

Reply:
Fixed